### PR TITLE
[Chronos] update chronos smape metric

### PR DESCRIPTION
**Pull request by rnwang04**
Make the smape metric in chronos the same with Orca.automl (https://github.com/intel-analytics/BigDL/issues/4766)

### CODE REVIEW

File: python/chronos/src/bigdl/chronos/metric/forecast_metrics.py [hunk 1]
```diff
@@ -15,10 +15,22 @@
...
 from numpy import ndarray
 from functools import partial
 from torchmetrics.functional import mean_squared_error, mean_absolute_error,\
-    mean_absolute_percentage_error, r2_score, symmetric_mean_absolute_percentage_error
+    mean_absolute_percentage_error, r2_score
+
+
+EPSILON = 1e-10
+
+
```
Comment (TheaperDeng) [inline python/chronos/src/bigdl/chronos/metric/forecast_metrics.py:27]: add some comments here to tell future developers why we need to implement this smape ourselves.

Comment (rnwang04) [reply python/chronos/src/bigdl/chronos/metric/forecast_metrics.py:27]: fix

**Review by TheaperDeng — APPROVED** [review]
LGTM

**Comment by rnwang04** [comment]
http://10.112.231.51:18888/job/BigDL-Chronos-PR-Validation/1001/
http://10.112.231.51:18888/job/Chronos-PR-Validation-Spark-2.4/1034/
http://10.112.231.51:18888/job/BigDL-PRNV-chronos-Python-Spark-2.4-py37-Horovod/1125/
http://10.112.231.51:18888/job/Chronos-PR-Validation-Spark-3.1/1029/

**Comment by TheaperDeng** [comment]
could you please run a full jenkins job for this pr?

**Comment by TheaperDeng** [comment]
to be merged

**Pull request merged**